MoreLinq.Test/MoreLinq.Test.csproj

```diff
@@ -0,0 +1,74 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <AssemblyTitle>MoreLinq.Test</AssemblyTitle>
+    <VersionPrefix>2.0.0</VersionPrefix>
+    <Authors>MoreLINQ Developers</Authors>
+    <TargetFrameworks>netcoreapp1.0;net451</TargetFrameworks>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <DebugType>portable</DebugType>
+    <AssemblyName>MoreLinq.Test</AssemblyName>
+    <OutputType>Exe</OutputType>
+    <PackageId>MoreLinq.Test</PackageId>
+    <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
+    <PackageTags>linq;extensions</PackageTags>
+    <PackageTargetFallback Condition=" '$(TargetFramework)' == 'netcoreapp1.0' ">$(PackageTargetFallback);dotnet5.4;portable-net451+win8</PackageTargetFallback>
+    <RuntimeFrameworkVersion Condition=" '$(TargetFramework)' == 'netcoreapp1.0' ">1.0.4</RuntimeFrameworkVersion>
```

Comment:
Unless I'm mistaken or [read it](https://github.com/dotnet/core/blob/master/release-notes/1.0/1.0.4.md) wrong (probably did as it's a mess anyways), 1.0.4 is the _runtime_ version whereas `.travis.yml` and `msbuild.sh` refer to the _SDK_ version that is 1.0.1.